charts/eks-anywhere-packages/templates/cronjob.yaml

```diff
@@ -29,20 +29,37 @@ spec:
                   valueFrom:
                     secretKeyRef:
                       name: aws-secret
-                      key: ID
-                      optional: true
+                      key: AWS_ACCESS_KEY_ID
                 - name: AWS_SECRET_ACCESS_KEY
                   valueFrom:
                     secretKeyRef:
                       name: aws-secret
-                      key: SECRET
-                      optional: true
+                      key: AWS_SECRET_ACCESS_KEY
                 - name: AWS_REGION
                   valueFrom:
                     secretKeyRef:
                       name: aws-secret
                       key: REGION
-                      optional: true
+                - name: REGISTRY_MIRROR_ENDPOINT
```

Comment:
Should these be optional values? Or are we ok with empty strings when nothing is passed.

Reply:
The secret gets created even though its fields are all empty, which allows customers to edit those secrets when they want to use such authentication.